website/docs/patterns/instantiate_objects/reference.md

```diff
@@ -4,40 +4,65 @@ title: Reference
 sidebar_label: Reference
 ---
 
-Hydra provides `hydra.utils.instantiate()` (and its alias `hydra.utils.call()`) for instantiating objects and calling functions. Prefer `instantiate` for creating objects and `call` for invoking functions.
+Hydra provides `hydra.utils.call()` (and its alias `hydra.utils.instantiate()`) for instantiating objects and calling functions. Prefer `instantiate` for creating objects and `call` for invoking functions.
 
 ```python
-def instantiate(config: Union[ObjectConf, DictConfig], *args: Any, **kwargs: Any) -> Any:
+def call(
+    config: Union[ObjectConf, TargetConf, DictConfig, Dict[Any, Any]],
+    *args: Any,
+    **kwargs: Any,
+) -> Any:
     """
-    :param config: An ObjectConf or DictConfig describing what to instantiate and what params to use
+    :param config: An object describing what to call and what params to use
     :param args: optional positional parameters pass-through
     :param kwargs: optional named parameters pass-through
-    :return: the return value from the specified class
+    :return: the return value from the specified class or method
     """
+    ...
+
+# Alias for call
+instantiate = call
 ```
 
-For using these functions, the config must have a key called `target`. If a key called `params` is also present, its value is passed as keyword arguments to class/function specified by `target`.
+For using these functions, the config must have a key called `_target_`. Any additional parameters are passed as keyword arguments to class/function specified by `target`.
 
 ### Example config
 ```yaml
 # target function name or class method fully qualified name
-target: foo.Bar
-# optional parameters dictionary to pass when calling the target
-params:
-  x: 10
+_target_: foo.Bar
+# additional parameters to pass when calling the target
+x: 10
 ```
 
 ### Example ObjectConf definition
-ObjectConf is defined in `hydra.types.ObjectConf`:
-```python
+TargetConf is defined in `hydra.types.TargetConf`:
+
+
+
+<div className="row">
+
+<div className="col col--6">
+
+```python title="Definition"
 @dataclass
-class ObjectConf(Dict[str, Any]):
-    # class, class method or function name
-    target: str = MISSING
-    # parameters to pass to target when calling it
-    params: Any = field(default_factory=dict)
+class TargetConf:
+    # class or function name
+    _target_: str = MISSING
+
 ```
 
+</div><div className="col col--6">
+
+```python title="Use it by subclassing and adding fields" 
+@dataclass
+class UserConf(TargetConf):
+    _target_ = "module.User"
```

Comment:
`_target_: str = "module.User"`

Reply:
Good catch. This is actually critical.
If the specified type does not match the one in the parent class it does not override properly in fucking @dataclasses.